aws-transfer-profile/pom.xml

```
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>software.amazon.transfer.profile</groupId>
+    <artifactId>aws-transfer-profile-handler</artifactId>
+    <name>aws-transfer-profile-handler</name>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <cfn.generate.args/>
+    </properties>
+
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/software.amazon.awssdk/transfer -->
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>transfer</artifactId>
+            <version>2.17.259</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/software.amazon.awssdk/sdk-core -->
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>sdk-core</artifactId>
+            <version>2.17.136</version>
+        </dependency>
```

While testing this resource type by registering it privately in my AWS account, I constantly got an error while trying to provision the profile. Following is the error I encountered:

```
Lambda function handler threw an uncaught exception: IS_NONE_AUTH_TYPE_REQUEST
```

Tried multiple things and ultimately, changing this dependency to be of same version as the transfer client worked.
```suggestion
        <dependency>
            <groupId>software.amazon.awssdk</groupId>
            <artifactId>sdk-core</artifactId>
            <version>2.17.259</version>
        </dependency>
```

---

Ahh thank you, good catch!